sample/src/main/res/values/dimens.xml

```diff
@@ -3,6 +3,6 @@
 
     <!-- Margins -->
     <dimen name="columnIntervals">8dp</dimen>
```

Comment:
I feel like columnSpacing is a better name for this, seems more commonly used for this kind of behaviour

Reply:
Renamed all Intervals to Spaces